src/java/org/apache/cassandra/tools/nodetool/layout/CassandraCliHelpLayout.java

```diff
@@ -0,0 +1,786 @@
+        {
+            CommandLine.Help help = entry.getValue();
+            CommandLine.Model.UsageMessageSpec usage = help.commandSpec().usageMessage();
+            String header = isEmpty(usage.header()) ? (isEmpty(usage.description()) ? "" : usage.description()[0]) : usage.header()[0];
+            table.addRowValues(colorScheme().commandText(entry.getKey()));
+
+            Ansi.Text leadingSpaces = Ansi.OFF.new Text(leadingSpaces(SUBCOMMANDS_INDENT), colorScheme());
+            table.addRowValues(leadingSpaces.concat(colorScheme().text(header)));
+
+            List<CommandLine.Model.OptionSpec> optionSpecs = help.commandSpec().options();
+            for (CommandLine.Model.OptionSpec optionSpec : optionSpecs)
+            {
+                if (optionSpec.hidden())
+                    continue;
+                table.addEmptyRow();
+
+                // Print the option description in multiple lines as it is set in the annotation.
+                for (int i = 0; i < optionSpec.description().length; i++)
+                {
+                    if (i == 0)
+                    {
+                        table.addRowValues(leadingSpaces.concat(
+                            colorScheme().optionText(String.format(SUBCOMMAND_OPTION_TEMPLATE,
+                                                                   optionSpec.longestName(),
+                                                                   colorScheme().text(optionSpec.description()[i])))));
+                    }
+                    else
+                    {
+                        table.addRowValues(leadingSpaces.concat(colorScheme().text(optionSpec.description()[i])));
+                    }
+                }
+            }
+        }
+        return table.toString();
+    }
+
+    @Override
+    public String footerHeading(Object... params)
+    {
+        return createHeading(FOOTER_HEADING, params);
+    }
+
+    @Override
+    public String footer(Object... params)
+    {
+        String[] footer;
+        if (commandSpec().parent() == null)
+            footer = isEmpty(commandSpec().usageMessage().footer()) ? new String[]{ USAGE_HELP_FOOTER } :
+                     commandSpec().usageMessage().footer();
+        else
+            footer = EMPTY_FOOTER;
+
+        TextTable table = TextTable.forColumns(colorScheme(), new Column(commandSpec().usageMessage().width(), 0, Column.Overflow.WRAP));
+        table.setAdjustLineBreaksForWideCJKCharacters(commandSpec().usageMessage().adjustLineBreaksForWideCJKCharacters());
+        table.indentWrappedLines = 0;
+
+        for (String summaryLine : footer)
+            table.addRowValues(String.format(summaryLine, params));
+        table.addEmptyRow();
+        return table.toString();
+    }
+
+    public String topLevelCommandListHeading(Object... params)
+    {
+        return createHeading(TOP_LEVEL_COMMAND_HEADING + "%n", params);
+    }
+
+    public String topLevelSynopsis(Object... params)
```

Comment:
`params` is unused?

Reply:
the same as https://github.com/apache/cassandra/pull/2497#discussion_r2232088557

This is a lambda that implements the `String render(Help help)` signature, fixed, but `help` is still unused.